docs/sources/flow/tasks/configure/configure-kubernetes.md

```diff
@@ -19,36 +19,53 @@ weight: 200
 
 # Configure {{% param "PRODUCT_NAME" %}} on Kubernetes
 
-To configure {{< param "PRODUCT_NAME" >}} on Kubernetes, perform the following steps:
+This page describes _how_ to apply a new configuration to {{< param "PRODUCT_NAME" >}} 
```

Comment:
Why emphasise `how` here?

Reply:
I can get rid of it, a leftover from previous version I had edited out. 

The essence here is that we only tell you **how to apply a configuration**. We don't tell you **what configuration to apply** and don't describe what to put inside of it. 

If there's a better way to put this, I'm open to suggestions!

Reply:
It was just the extra emphasis on the word "how". I'd remove the italics here and just leave the word `how`... but it's OK either way.